src/sql/src/statements/create.rs

```diff
@@ -218,6 +218,13 @@ pub struct CreateExternalTable {
     pub engine: String,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Visit, VisitMut)]
+pub struct CreateTableLike {
+    /// Table name
+    pub name: ObjectName,
+    pub target: ObjectName,
```

Comment:
```suggestion
    /// The target table name
    pub target: ObjectName,
```

Reply:
The field name is a little bit confusing. `name` is actually the name of table to create, while `target` is where the schema comes from.